subsys/bluetooth/audio/pacs.c

```diff
@@ -450,7 +450,7 @@ BT_GATT_SERVICE_DEFINE(pacs_svc,
 		      BT_GATT_PERM_READ_ENCRYPT,
 		      snk_loc_read, NULL, NULL),
 #endif /* CONFIG_BT_PAC_SNK_LOC_WRITEABLE */
-	BT_AUDIO_CCC(snk_loc_cfg_changed),
+	IF_ENABLED(CONFIG_BT_PAC_SNK_LOC, (BT_AUDIO_CCC(snk_loc_cfg_changed))),
```

Comment:
maybe refactor this to (BT_PAC_SNK_LOC_WRITEABLE depends on BT_PAC_SNK_LOC anyway):

#if defined(CONFIG_BT_PAC_SNK_LOC)
#if defined(CONFIG_BT_PAC_SNK_LOC_WRITEABLE)
  //writeable
#else
  // non-writeable
#endif
BT_AUDIO_CCC(snk_loc_cfg_changed),
#endif